src/lib/blueprint.ts

```diff
@@ -2,14 +2,109 @@ import type { Openapi } from './openapi.js'
 
 export interface Blueprint {
   name: string
+  routes: Route[]
 }
 
+interface Route {
+  name: string
+  path: string
+  description: string | null
```

Comment:
We should require all routes have a description and default to `''`.

```suggestion
  description: string
```

We could emit warnings to the console things that are missing this, e.g.,

```
if (description.trim().length === 0) console.warn(`... has an empty description`